src/components/calculator.js

```diff
@@ -16,6 +16,10 @@ import { useState } from 'react';
       };
     return (
       <div className='calculator-container'>
+        <div>
+        <h1>Lets do some math.</h1>
+      </div>
+      <div className="calc-div">
         <div className='result-pad'>{(total ? total : '') + (operation ? operation : '') + (next ? next : '')}</div>
         <div className='buttons'>
             <button type='button' onClick={handleEvent}>AC</button>
```

Comment:
[OPTIONAL] Every time I use the multiplication (*) sign on your calculator the entire page goes blank. Kindly look into to make sure that all arithmetic operations work without disturbing the webpage.